TypeCobol/Compiler/Symbols/AbstractScope.cs

```diff
@@ -9,6 +9,9 @@
 
 namespace TypeCobol.Compiler.Symbols
 {
+    /// <summary>
+    /// Don't confuse with Scope class
```

Comment:
Maybe a little bit more of an explanation 😄 ? "Represents any symbol that contain other symbols (i.e. ProgramSymbol or NamespaceSymbol)"

Reply:
See last changes